eregs_extensions/epa_regparser/term_defs/__init__.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+
+term_defs = {
+    "264": [
+        ("CROMERR Costs",
+         "CROMERR Costs")
```

Comment:
Should the second element in this tuple include more context? Otherwise _everywhere_ that "CROMERR Costs" appears, we'll be "defining" a term.